style1.css

```diff
@@ -0,0 +1,98 @@
+*{
+  padding: 0;
+  margin: 0;
+}
+body, html{
+  overflow-x: hidden;
+}
+nav{
+  margin: 0 auto;
+  max-width: 1100px;
+  width: 100%;
+}
+#checkbox-menu,
+.toggle{
+  display:none;
+}
+.menu{
+  padding: 0;
+  margin: 0;
+  height: 50px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  list-style-type: none;
+}
+.menu li a{
+  color: #000;
+  text-decoration: none;
+  align-self: center;
+  font-size: 13px;
+  padding: 10px 15px;
+  transition: background .2s linear;  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/media_adaptive/pullRequest?prid=5498104)